common/utils.js

```diff
@@ -453,6 +453,19 @@ const getItsEdgeContract = (chainConfig) => {
     return itsEdgeContract;
 };
 
+function getAllChains(config) {
+    const allChains = Object.keys(config.chains);
+    return allChains;
+}
+
+function getITSChains(config) {
+    const allChains = getAllChains(config);
+    return allChains.filter(
+        (chain) =>
+            config.chains[chain].contracts?.InterchainTokenService && config.chains[chain].contracts?.InterchainTokenService?.address,
```

Comment:
```suggestion
            config.chains[chain].contracts?.InterchainTokenService?.address,
```
just this is enough due to `?` operator